packages/headless/src/controllers/core/breadcrumb-manager/headless-core-breadcrumb-manager.ts

```diff
@@ -204,6 +205,10 @@ export const getBreadcrumbs = <T extends AnyFacetSetState>(
     facetId: string;
     selection: InferFacetSliceValueType<T>;
   }) => void,
+  executeToggleExclude: (payload: {
+    facetId: string;
+    selection: InferFacetSliceValueType<T>;
+  }) => void,
```

Comment:
There's starting to be quite a few properties here, I think it would be easier to read if we replaced these properties with an object/interface.

Reply:
I mostly agree, but I messed around with this a fair amount and couldn't seem to resolve the typing everywhere. The main problem has to do with the `executeToggleSelect/Exclude` methods, or more specifically, the `selection` parameter for these methods. No matter what I changed, I couldn't seem to please the parser for these parameters. 

Also the further I got into these changes, I noticed that it didn't really "clean" things up as much as I would hope as it would still be the same amount of parameters, but now nested within an object that needed to be further typed because of generics...

I could be misunderstanding and/or incorrectly implementing what you're suggesting though, so I'd be happy to talk about this further on Slack if you really want this change?

Reply:
tldr of the discussion? 🍿

Reply:
No discussion happened 😅 @btaillon you're welcome to chime in if you'd like (or ping me on Slack), but if it's not something you feel strongly enough about, I'll go ahead and merge hehe

Reply:
Oh, I just meant refactoring the parameters of `getBreadcrumbs` to take an object instead of ordered parameters. It shouldn't affect validation or anything like that.

Reply:
I'm still not certain if I love having it this way over how it was, but I move it to its own type! Let me know if this is what you had in mind 🙂